src/sys/ptrace.rs

```diff
@@ -134,8 +134,8 @@ pub unsafe fn ptrace(request: Request, pid: Pid, addr: *mut c_void, data: *mut c
     }
 }
 
-fn ptrace_peek(request: Request, pid: Pid, addr: *mut c_void, data: *mut c_void) -> Result<c_long> {
-    let ret = unsafe {
+unsafe fn ptrace_peek(request: Request, pid: Pid, addr: *mut c_void, data: *mut c_void) -> Result<c_long> {
+    let ret = {
```

Comment:
There's no reason to have this in braces, just make these two separate lines instead.

Reply:
fixed that and reformatted to avoid exceeding 80 columns of length.